lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -25,6 +26,17 @@ function requestHandler(request, response) {
   const queryString = requestUrl.search && parseQueryString(requestUrl.search.slice(1));
   let absoluteFilePath = path.join(__dirname, filePath);
 
+  if (filePath === '/') {
+    const fixturePaths = glob.sync('**/*.html', {cwd: __dirname});
+    const html = `
+      <html>
+      <h1>Smoke test fixtures</h1>
+      ${fixturePaths.map(p => `<a href=${p}>${p}</a>`).join('<br>')}
+    `;
+    sendResponse(200, html);
```

Comment:
```js
const html = `
  <html>
  <h1>Smoke test fixtures</h1>
  ${fixturePaths.map(p => `<a href="${encodeURI(p)}">${escape(p)}</a>`).join('<br>')}
`;
response.writeHead(200, {
  'Content-Security-Policy': `default-src 'none';`,
});
finishResponse(html);
```

let's be safe :)

Reply:
now i'm wondering what sort of js payloads you can inject using just file paths